roles/k8s-node/tasks/main.yml

```diff
@@ -1,8 +1,40 @@
-- name: Disable SELinux
-  selinux:
-    state: disabled
-
-- name: Run the node-e2e tests
-  command: ginkgo --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]" --nodes=8 /usr/local/bin/e2e_node.test -- --k8s-bin-dir /usr/local/bin --test.timeout=1h5m0s --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --cgroup-driver={{ cgroup_driver }}"
-  args:
-    chdir: /tmp
+- name: change systemd cgroup driver to default cgroupfs
+  replace:
+    path: /etc/containerd/config.toml
+    regexp: 'SystemdCgroup = true'
+    replace: 'SystemdCgroup = false'
+
+- name: Restart containerd service
+  systemd:
+    name: containerd
+    state: restarted
+
+- name: Install prereq packages
+  yum:
+    name: "{{ packages }}"
+    state: present
+  vars:
+    packages:
+      - git
+      - make
+      - gcc
+
+- name: Clone a kubernetes github repo
+  git:
+    repo: https://github.com/kubernetes/kubernetes
```

Comment:
wondering how do we mention the version it is suppose to clone? may be checkout with branch like main/master or any released version

Reply:
I actually included the task of downloading k8s bits to the `k8s-node-remote.yml` playbook and thought could pass on the `--build-version` flag to kubetest2 command.

My fault, that I overlooked `make` command would build the binaries for the cloned Kubernetes repo.

So, I shall now add the step to switch to the required commit to this task `Clone a kubernetes github repo`

We need the binaries of version at https://storage.googleapis.com/k8s-release-dev/ci/latest.txt as we use in this job [here](https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml#L95)
for example, if version is `v1.29.0-alpha.3.131+8509ab82b96c`,
`git checkout 8509ab82b96caa` should work.

```
[root@raji-workspace2 ~]# git clone https://github.com/kubernetes/kubernetes
Cloning into 'kubernetes'...
remote: Enumerating objects: 1492149, done.
remote: Counting objects: 100% (188/188), done.
remote: Compressing objects: 100% (104/104), done.
remote: Total 1492149 (delta 91), reused 125 (delta 84), pack-reused 1491961
Receiving objects: 100% (1492149/1492149), 969.77 MiB | 16.39 MiB/s, done.
Resolving deltas: 100% (1095605/1095605), done.
Updating files: 100% (24946/24946), done.
[root@raji-workspace2 ~]# cd kubernetes
[root@raji-workspace2 kubernetes]# git checkout 8509ab82b96caa
Note: switching to '8509ab82b96caa'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 8509ab82b96 Merge pull request #121851 from andyzhangx/fix-smb-IsCorruptedMnt
[root@raji-workspace2 kubernetes]#